Sources/ClientModels/TurnBasedMatchData.swift

```diff
@@ -28,7 +28,7 @@ public struct TurnBasedMatchData: Codable, Equatable {
     public var playerIndexToId: [SharedModels.Move.PlayerIndex: SharedModels.Player.Id]
 
     public init(
-      playerIndexToId: [SharedModels.Move.PlayerIndex: SharedModels.Player.Id] = [:]
+      playerIndexToId: [SharedModels.Move.PlayerIndex: SharedModels.Player.Id]
```

Comment:
Figure it's better to be explicit so we don't accidentally clear out old mappings.